scripts/index.js

```diff
@@ -1 +1,95 @@
-// add your solutions here
\ No newline at end of file
+// Positive Numbers
+
+function returnsPositiveNumbers(givenArray) {
+    newArray = [];
+    const positiveNumbers = givenArray.filter(num => {
+        return num > 0;
+    })
+    newArray.push(positiveNumbers);
+    return newArray;
+}
+
+// Even Numbers
+
+function returnEvenNumbers(givenArray) {
+    newArray = [];
+    const evenNumbers = givenArray.filter(num => {
+        return num % 2 === 0;
+    })
+    newArray.push(evenNumbers);
+    return newArray;
+}
+
+// Square the Numbers
+function squareTheNumbers(givenArray) {
+    newArray = [];
+    let squaredNumbers = givenArray.map(num => {
+        return num * num;
+    })
+    newArray.push(squaredNumbers);
+    return newArray;
+}
+
+// Cities 1
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 }
+  ];
+
+function nameColdCities(givenArray) {
+    newArray = [];
+    givenArray.forEach(function (value) {
+        if (value.temperature < 70)
+        newArray.push(value.temperature);
+    })
+    return newArray;
+}
+
+// Good job
+
+var people = [
+    'Dom',
+    'Lyn',
+    'Kirk',
+    'Autumn',
+    'Trista',
+    'Jesslyn',
+    'Kevin',
+    'John',
+    'Eli',
+    'Juan',
+    'Robert',
+    'Keyur',
+    'Jason',
+    'Che',
+    'Ben'
+  ];
+
+  people.forEach(function (name) {
+      console.log(`Good job, ${name}!`)
+  });
+
+
+// 3 times
+function sayHello (words) {
+    console.log("Hello, World!");
+}
+function call3Times(fun) {
+    fun(sayHello);
+    fun(sayHello);
+    fun(sayHello);
+  }
```

Comment:
I couldn't get this one to run and I think you have it a little inside-out? Look at this solution and see if it makes sense: 
`function speak() {`
 `   console.log("Hello, world!")`
`}`
`function call3Times(fun) {`
`    fun();`
`    fun();`
`    fun();`
`}`
you can tell that `fun()` is a function because of the `()`